performance-comparisons/v11.10/results/report_0_postgresql_bounded_6500.md

```
@@ -0,0 +1,711 @@
+### Store times avg (worsened):
+| | | Base | Actual | Delta | Delta % |
```

markdownlint-cli2 (0.23.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)